### ISSUE TITLE
Can't open or install extensions due to permissions

### ISSUE THREAD

**Issue by JoshuaA9088** [Microsoft/vscode#54421]
- VSCode Version: 1.25.1
- OS Version: Ubuntu 16.04

Steps to Reproduce:

1. Fresh install
2. Try to launch from terminal with `--verbose`

Does this issue occur when all extensions are disabled?: N/A

I noticed when fresh installing VS code I had to manually change the permissions of `/home/$USER/.config/Code/` to launch and `/home/$USER/.vs/` to install extensions. Otherwise I get `Error: EACCES: permission denied`  Not sure if this is an isolated issue, although I have had this happen on 2 of my ubuntu machines.

**Comment by bpasero** [comment]
Maybe an installation issue?

**Comment by Tyriar** [comment]
I've seen this before when sudo is used to launch the first time, @JoshuaA9088 do you think that could have happened?

**Comment by JoshuaA9088** [comment]
I didn't run with sudo when this happened. It may have been because I bundle installed it with anaconda3. However, manually reinstalling didn't fix the issue.

**Comment by Tyriar** [comment]
@JoshuaA9088 yes I'd guess it's related to the anaconda build then, you can try reporting the issue to them. Since you've already worked around the problem I'll close this off.
